xgcm/test/test_exchange.py

```diff
@@ -130,6 +130,7 @@ def test_create_periodic_grid(ds):
         assert connect_right[2] is False
 
 
+@pytest.mark.skip(reason="We shouldnt test private functions")
```

Comment:
This will not work by design (the private function should be removed as part of this PR), so how do we best deal with this? I think skipping it for now might be good?